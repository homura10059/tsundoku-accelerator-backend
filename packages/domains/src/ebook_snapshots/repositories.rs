pub mod db;
pub mod scraper;
