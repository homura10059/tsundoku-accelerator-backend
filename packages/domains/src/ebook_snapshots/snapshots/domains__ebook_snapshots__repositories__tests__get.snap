---
source: packages/domains/src/ebook_snapshots/repositories.rs
expression: actual.payment_ebook
---
Some(
    Payment {
        price: "3344",
        points: "152",
    },
)
